.github/workflows/build.yml

```
@@ -0,0 +1,31 @@
+name: Build
+
+on:
+  push:
+    branches: [ 'main' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20.x'
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v2
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
```

Based on their [doc](https://github.com/golangci/golangci-lint-action#how-to-use) it seems that `version` is mandatory. Furthermore it seems that the action is expecting `.golangci.yml` at the root of the repository but it is not clear to me if that MUST exist or not.

```suggestion
        uses: golangci/golangci-lint-action@v3
        with:
          version: v1.54
```